site/sfguides/src/mfa_audit/mfa_audit.md

```diff
@@ -0,0 +1,189 @@
+author: Chanin Nantasenamat
+id: mfa-audit
+summary: Learn how to build a Streamlit app in Snowflake Notebooks to audit and monitor Multi-Factor Authentication (MFA) status of users, with automated email notifications for non-compliant users.
+categories: featured, cybersecurity, data-engineering
+environments: web
+status: Published
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues
+tags: Streamlit, Cybersecurity, MFA, Email Notifications
+
+# Building an MFA Audit System with Streamlit in Snowflake Notebooks
+<!-- ------------------------ -->
+## Overview
+Duration: 2
+
+Learn how to create an interactive Multi-Factor Authentication (MFA) audit system using Streamlit in Snowflake Notebooks. This application helps security administrators monitor user MFA compliance and automatically notify relevant stakeholders about users who haven't enabled MFA.
+
+### What You'll Learn
+- How to create and query a user dataset containing MFA status
+- How to set up email notifications in Snowflake
+- How to build an interactive Streamlit interface for MFA monitoring
+- How to implement automated email reporting for non-compliant users
+
+### What You'll Build
+An interactive Streamlit application that displays user MFA status and sends formatted email reports to system administrators about users who haven't enabled MFA.
+
+### Prerequisites
+- Access to Snowflake Notebooks
+- System administrator privileges in Snowflake
+- Basic understanding of SQL and Python
+- Valid email address for receiving notifications
+
```

Comment:
@sfc-gh-cnantasenamat Add a new sub-section "What You'll Need" and add a link to Snowflake account signup/signin page. Like so "Access to a [Snowflake account](https://signup.snowflake.com/) account."